extensions/wikia/EmbeddableDiscussions/EmbeddableDiscussionsController.class.php

```diff
@@ -87,7 +90,8 @@ private function checkArguments( array $args, $modelData, &$errorMessage ) {
 		// category must be a valid category
 		if ( $modelData['invalidCategory'] ) {
 			$errorMessage = wfMessage( 'embeddable-discussions-parameter-error', $args['catid'],
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "catid" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)